README.md

```diff
@@ -1,5 +1,6 @@
 [![Maintainability](https://api.codeclimate.com/v1/badges/0f12a7e73736f8bbfb9d/maintainability)](https://codeclimate.com/github/Open-CMSIS-Pack/vscode-cmsis-debugger/maintainability)
 [![Test Coverage](https://api.codeclimate.com/v1/badges/0f12a7e73736f8bbfb9d/test_coverage)](https://codeclimate.com/github/Open-CMSIS-Pack/vscode-cmsis-debugger/test_coverage)
+[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/github.com/Open-CMSIS-Pack/vscode-cmsis-debugger/badge)](https://securityscorecards.dev/viewer/?uri=github.com/Open-CMSIS-Pack/vscode-cmsis-debugger)
```

Comment:
Could you please update https://github.com/Open-CMSIS-Pack/vscode-cmsis-debugger/blob/7a0101464abb0565b3205f842bb05bbd39abb162/.github/workflows/ci.yml#L57  to filter out the badge before packaging the extension VSIX? I find them rather irritating when showing inside the VS Code extension manager.

Reply:
Good catch 👍🏽 
Addressed : [Remove badge](https://github.com/Open-CMSIS-Pack/vscode-cmsis-debugger/pull/220/commits/045410af447247f91587b51da5fcf449ff201e9a)

Reply:
Can confirm the badge is not in the VSIX files coming from CI.